### DISCUSSION TITLE
Discord Bot connected to the Bing API

### DISCUSSION THREAD

**Discussion by KyoshiHikari** [acheong08/EdgeGPT#292] [Ideas]
Hello dear people,

I tried to make EdgeGPT as easy as possible with python and I had the idea to make a discord bot with it,
If you want, I can share the code snippet with you, for future things so you can implement it into your own discord bot.
Are you interested? If not, can i share my code on my repo with the implementation of EdgeGPT?

Regards
xkiiyoshiix

**Comment by codelauncher444** [comment]
Is the code working currently? There is an issue with mine.

**Reply by KyoshiHikari** [reply]
Yes the code is working currently.
What is the problem with your's?

**Comment by codelauncher444** [comment]
The system did not work today, although I renewed the cookies.
File "C:\Users\90533\AppData\Local\Programs\Python\Python311\Lib\site-packages\EdgeGPT.py", line 255, in init
raise NotAllowedToAccess(self.struct["result"]["message"])
EdgeGPT.NotAllowedToAccess: Sorry, you need to login first to access this service.

Is that happening to everyone? If not, can someone help me to solve this issue?

**Reply by KyoshiHikari** [reply]
The indicated error occures, when youre not allowed to use Bing Chat GPT so you need an account that have access to it.

**Reply by codelauncher444** [reply]
what type of an account do you mean? Btw, I have just seen that in the requirments, Chinese mainland VPN is required. Not related with this, do you have any idea about the limitations of usage in Edge GPT

**Reply by KyoshiHikari** [reply]
You need to save the cookie from your account to use the bot.
